modules/specifying-infrastructure-vsphere-failure-domains.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+// * post_installation_configuration/sphere-failure-domain-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="specifying-infrastructure-vsphere-failure-domains_{context}"]
+= Specifying multiple failure domains for your cluster on VSphere
+
+You can configure the `infrastructures.config.openshift.io` configuration resource to specify multiple failure domains for your {product-title} cluster that runs on a VMware VSphere instance.
+////
+OCP cluster or VMware vSphere cluster
```

Comment:
Hi @rvanderp3 . I added a comment here to clarify if I should state OCP cluster or VMWare vSPhere cluster. I assume it's OCP cluster as this is what needs to run on the vSPhere platform, but maybe in a sense this cluster is technically a VMware vSphere cluster as it runs on the vSphere cluster?